jax/_src/random.py

```diff
@@ -1724,3 +1724,85 @@ def ball(
   g = generalized_normal(k1, p, (*shape, d), dtype)
   e = exponential(k2, shape, dtype)
   return g / (((jnp.abs(g) ** p).sum(-1) + e) ** (1 / p))[..., None]
+
+def vonmises(
+  key: KeyArray,
+  x: float,
+  kappa: float,
+  shape: Shape = (),
```

Comment:
Other distributions allow arbitrarily-sized arrays for the distribution parameters: see how `gamma` treats the `a` parameter, for example. We should follow that convention here.